Core.DomainModel/ItSystemUsage/ItSystemUsage.cs

```diff
@@ -974,6 +980,36 @@ public ItSystemUsageValidationResult CheckSystemValidity()
             return new ItSystemUsageValidationResult(errors);
         }
 
+        public Maybe<OperationError> AddPersonalData(GDPRPersonalDataOption option)
+        {
+            if (SensitiveDataLevels.Any(x => x.SensitivityDataLevel == SensitiveDataLevel.PERSONALDATA) == false)
```

Comment:
Make a helper 
```
private boole HasSensitivityLevel(SensitiveDataLevel level) ...

```
and use that. Much easier to read and is needed in the "remove" case as well ... as well as in your "conflicts check" ( The add/remove sensitivity level)... will clean up a lot of code

Reply:
Og it actually exists... use it:
![image](https://user-images.githubusercontent.com/13116889/207254307-9c21ca40-6783-43bd-9b0c-8adca9523ba1.png)